go/api/base_client.go

```diff
@@ -30,6 +30,13 @@ type BaseClient interface {
 
 const OK = "OK"
 
+var (
```

Comment:
We need to find a way to make this immutable

Reply:
Replaced this with functions.